src/OFS/ObjectManager.py

```diff
@@ -125,6 +125,10 @@ def checkValidId(self, id, allow_dup=0):
         raise BadRequest(
             'The id "%s" is invalid because it '
             'ends with two underscores.' % id)
+    if id.startswith('@@') or id.startswith('++'):
+        raise BadRequest(
+            'The id "%s" is invalid because it starts with characters '
+            'reserved for Zope views lookup.' % id)
```

Comment:
```suggestion
            "reserved for Zope's view lookup." % id)
```

Reply:
Das ist eine Verschlimmbesserung, geht nicht.

Reply:
Sorry, maybe I did not understand the intent of the sentence of the exception.

Reply:
It's a matter of better English